common/data/src/index.ts

```diff
@@ -30,24 +30,21 @@ export function schemaToGlueColumns(jsonSchema: JsonSchema): glue.Column[] {
   })
 }
 
-export type EventTableColumn = {
+export type EventTableSchema = {
   chain: string
   network: string
   provider: string
   type: string
+  height: number
+  block: string
+  transaction: string
   created_at: string
   address: string
-  height: number
   to_address: string
-  candidate: string
-  candidate_list: string[]
+  validator: string
+  validator_list: string[]
```

Comment:
Thoughts making most of these optional and then including them only when needed? When trying to make sense of the data for querying, it may be clearer to make columns don't apply to a particular row empty (as in the column is not used for that particular event type). For the specific example of `validator_list`, there would be a conventional difference between an empty array and a null value.

Reply:
Sounds good, I updated the schema for both.